changelog/std-algorithm-searching-extremum.dd

```diff
@@ -0,0 +1,21 @@
+Performance improvements for `std.algorithm.searching.{min,max}Element`
+
+$(REF minElement, std, algorithm, searching) and $(REF maxElement, std, algorithm, searching)
+are now a lot faster as a special path for the identity case is provided.
```

Comment:
"a lot" -> "considerably"

better yet:

"considerably (almost 2x in microbenchmarks)" (or whatever)

i.e. we're trying to avoid colloquialisms